frontend/package.json.orig

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "nextjs",
+  "version": "3.0.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^3.9.2",
+    "@material-ui/icons": "^3.0.2",
+    "@material-ui/styles": "^3.0.0-alpha.10",
+    "apollo-boost": "^0.1.27",
+    "apollo-client": "^2.4.3",
+    "flow-typed": "^2.5.1",
+    "graphql": "^14.1.1",
+    "graphql-tag": "^2.10.1",
+    "graphql-tools": "^4.0.4",
+    "isomorphic-unfetch": "^3.0.0",
+    "jss": "^9.8.7",
+    "next": "^7.0.2",
+    "prop-types": "^15.6.2",
+    "react": "^16.8.0",
+    "react-apollo": "^2.4.1",
+<<<<<<< HEAD
```

Comment:
Is this working? Probably going to launch a syntax error.

Reply:
should remove this file, is the .origin file resulting from merge